src/mathMLTree.js

```diff
@@ -189,15 +189,15 @@ class SpaceNode implements MathDomNode {
         // for a table of space-like characters.  We use Unicode
         // representations instead of &LongNames; as it's not clear how to
         // make the latter via document.createTextNode.
-        if (width >= 0.05555 && width <= 0.05556) {
+        if (width >= 0.0555 && width <= 0.0556) {
             this.character = "\u200a";           // &VeryThinSpace;
         } else if (width >= 0.1666 && width <= 0.1667) {
             this.character = "\u2009";           // &ThinSpace;
         } else if (width >= 0.2222 && width <= 0.2223) {
             this.character = "\u2005";           // &MediumSpace;
         } else if (width >= 0.2777 && width <= 0.2778) {
             this.character = "\u2005\u200a";     // &ThickSpace;
-        } else if (width >= -0.05556 && width <= -0.05555) {
+        } else if (width >= -0.0556 && width <= -0.0555) {
```

Comment:
nit: these numbers probably don't need rounding since they don't end up in the markup we produce.

Reply:
No problem. I've reverted these numbers.